src/tasks/components/TaskRunsRow.tsx

```diff
@@ -52,6 +52,7 @@ class TaskRunsRow extends PureComponent<Props, State> {
             color={ComponentColor.Default}
             text="View Logs"
             onClick={this.handleToggleOverlay}
+            style={{ marginRight: 10 }}
```

Comment:
I tried doing that but it kept getting overwritten. the only way i could actually apply the style i need is by using inline styling

Reply:
super weird. It might be worth checking to see where it's being overwritten and then being super specific with applying the class, so like:

```
.class-1 {
   .class-2 {
      .your_class {
         ...styles here
      }
   }
}
```